src/types/tracker.ts

```diff
@@ -3,34 +3,73 @@ import type { StepConfig } from './steps'
 
-type MappedEventNames =
-  | 'screen_welcome'
-  | 'screen_userConsent'
-  | 'screen_document_front_capture_file_upload'
-  | 'screen_document_front_confirmation'
+export type LegacyTrackedEventNames =
+  | 'screen_complete'
+  | 'screen_complete_crossdevice_mobile_success'
+  | 'screen_document_country_select'
+  | 'screen_document_crossDevice_client_intro'
+  | 'screen_face_crossDevice_client_intro'
+  | 'screen_poa_crossDevice_client_intro'
+  | 'screen_crossDevice_desktop_submit'
+  | 'screen_crossDevice_crossdevice_link'
+  | 'screen_crossDevice'
+  | 'copy link selected'
+  | 'qr code selected'
+  | 'sms selected'
+  | 'screen_crossDevice_mobile_notification_sent'
+  | 'screen_crossDevice_mobile_connected'
+  | 'screen_crossDevice_sms_failed'
+  | 'Starting upload'
+  | 'screen_document_back_capture_camera_error'
+  | 'screen_document_front_capture_camera_error'
+  | 'Taking live photo of document'
   | 'screen_document_back_capture_file_upload'
+  | 'screen_document_back_capture'
+  | 'screen_document_front_capture_file_upload'
+  | 'screen_document_front_capture'
   | 'screen_document_back_confirmation'
+  | 'screen_document_front_confirmation'
+  | 'screen_document_fallback_clicked'
+  | 'screen_document_image_quality_guide'
+  | 'screen_document_type_select'
+  | 'screen_document_document_video_capture_file_upload'
+  | 'screen_document_document_video_capture'
+  | 'screen_document_document_video_capture_fallback_triggered'
   | 'screen_face_selfie_intro'
+  | 'screen_face_selfie_capture_camera_error'
+  | 'screen_face_selfie_capture_file_upload'
   | 'screen_face_selfie_capture'
   | 'screen_face_selfie_confirmation'
-  | 'screen_face_video_intro'
-  | 'screen_face_video_capture_step_1'
-  | 'screen_face_video_capture_step_2'
-  | 'screen_document_type_select'
-  | 'screen_document_country_select'
-  | 'screen_crossDevice'
-  | 'screen_crossDevice_crossdevice_link'
-  | 'Starting upload'
-
-export type TrackedEventNames =
-  | MappedEventNames
-  | 'started flow'
-  | 'Completed upload'
+  | 'screen_face_selfie_capture_fallback_triggered'
   | 'Snapshot upload completed'
-  | 'Starting live photo upload'
   | 'Starting snapshot upload'
-  | 'Taking live photo of document'
+  | 'Starting live photo upload'
+  | 'screen_face_face_video_capture_camera_error'
+  | 'screen_face_face_video_capture_file_upload'
+  | 'screen_face_face_video_capture'
+  | 'screen_face_video_capture_step_1'
+  | 'screen_face_video_capture_step_2'
+  | 'screen_face_video_challenge_load_failed'
+  | 'screen_face_video_challenge_loaded'
+  | 'screen_face_video_challenge_requested'
+  | 'screen_face_face_video_confirmation'
+  | 'screen_face_face_video_capture_fallback_triggered'
+  | 'screen_face_video_intro'
   | 'completed flow'
+  | 'started flow'
+  | 'screen_forbidden_client_error'
+  | 'screen_generic_client_error'
+  | 'screen_interrupted_flow_error'
+  | 'screen_poa_poa_file_upload'
+  | 'screen_poa_poa'
```

Comment:
Are these instances of duplicated `poa_poa`, `face_face` in the `LegacyTrackedEventNames` a typo or side effect of the way we're constructing the event names in the existing/legacy tracking functionality?